src/Components/App/App.js

```diff
@@ -1,19 +1,43 @@
 import "./App.css"
+import React, { useEffect, useState } from "react"
 import { Route, Routes, NavLink } from "react-router-dom"
 import { useSelector, useDispatch } from "react-redux"
-import { increment, decrement } from "../../actions"
-import Header from "../../Components/Header/Header"
+import { assignUsers, assignTrails } from "../../actions"
+import Header from "../Header/Header"
 import Welcome from "../Welcome/Welcome"
 import AboutUs from "../AboutUs/AboutUs"
 import SavedTrails from "../SavedTrails/SavedTrails"
 import IndividualTrail from "../IndividualTrail/IndividualTrail"
 import AllTrails from "../AllTrails/AllTrails";
+import Login from "../Login/Login"
 import { Form } from "react-router-dom";
 
-function App() {
-  const counter = useSelector((state) => state.counter)
+const App = () => {
+  const users = useSelector((state) => state.users)
   const dispatch = useDispatch()
 
+  const getData = (url) => {
+    return fetch(url, {
+    })
+      .then(response => response.json())
+      .catch(err => console.log(err))
+  }
```

Comment:
This looks like a good function to have in a `utilities` directory of some kind. Some sort of shared area where you can have generic functions that are reusable in multiple files.

Helps make the code more DRY but not having to rewrite things.